### DISCUSSION TITLE
Where did the issues tab go?

### DISCUSSION THREAD

**Discussion by MPLew-is** [krzyzanowskim/CryptoSwift#969] [General]
There used to be a tab containing issues tracking bugs and feature requests, was that purposely removed or is it something that can be brought back? There were a number of things I was tracking in there, and they just seem to have vanished in the last few weeks.

**Comment by MPLew-is** [comment]
It looks like a lot of discussions might have been converted from issues, but it doesn't looks like closed issues made it over? There were a lot of conversations with workarounds and statuses (like PBKDF2 debug build performance) that have just disappeared.

**Reply by krzyzanowskim** [reply]
I moved some issues here. I will look for the ones you're talking about to move it here

**Reply by krzyzanowskim** [reply]
I did move a few more old issues here. most likely something you've been looking for @MPLew-is

**Reply by MPLew-is** [reply]
Yep, that brought back the one I was looking for - thank you!

**Comment by krzyzanowskim** [comment]
Issues tab is gone. Discussion is here. mainly because many issues were more like a comment than a bug report.
